internal/handlers/commonparams/extract_params_test.go

```diff
@@ -28,7 +28,7 @@ import (
 func TestParse(t *testing.T) {
 	type allTagsThatPass struct { //nolint:vet // it's a test struct
 		DB           string          `ferretdb:"$db"`
-		Collection   string          `ferretdb:"collection"`
+		Collection   string          `ferretdb:"find,collection"`
```

Comment:
Could you add a test case where `Collection` is `*type.Document` please? For example, `explain` is one of those case :hugs: 

> The explain command expects the command document to be passed as the command parameter.
```
db.runCommand(
    {
        explain: {
        "update": {...}
        }
    }
)
```